assets/app/components/site/pageListItem.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+const propTypes = {
+  page: React.PropTypes.object
+};
+
+class PageListItem extends React.Component {
+  getButtonCopy(page) {
+    return page.type === 'dir' ? 'Open' : 'Edit';
+  }
+
+  render() {
+    const { page } = this.props;
+
+    return (
+      <li className="list-group-item">
+        <div className="usa-grid">
+          <div className="usa-width-two-thirds">
+            {page.name}
+          </div>
+          <div className="usa-width-one-third">
+            <a className="usa-button usa-button-outline file-list-item-button" href={page.url}>
```

Comment:
I think we'll want to change `href` here. In the current application, we basically have a route like `site/:id/edit/:branch(/)*file` where `file` is the value of `path` in the Github API response.

In order to do that though, we'll probably need to pass more of the state all the way down, as we'll need the site id and branch. Is this something that would be easier to do once we have react-router in place?